lib/ansible/plugins/inventory/gitlab_runners.py

```diff
@@ -31,11 +31,15 @@
               - gitlab_runners
         server_url:
             description: The URL of the GitLab server, with protocol (i.e. http or https).
+            env:
+              - name: GITLAB_SERVER_URL
```

Comment:
```suggestion
              - name: GITLAB_SERVER_URL
                version_added: '2.10'
```
Same for the other option.